src/standalone/Vegetation/Canopy.jl

```diff
@@ -322,7 +321,7 @@ has a single update_aux! function, given here.
 function ClimaLSM.make_update_aux(
     canopy::CanopyModel{
         FT,
-        <:BeerLambertModel,
+        <:AbstractRadiationModel,
```

Comment:
I think we can either keep this as <:AbstractRadiationModel, but then specify in the docs for AbstractRadiationModel that it must have field `parameters` with \Omega, \lambda_gamma, and ld, since these are unpacked from it in this function. 

We could also update the methods for `auxiliary_variables` and `auxiliary_types` in radiation.jl to be for AbstractRadiationModel instead of having the same behavior but defined in two methods for each concrete model type. This would reduce some duplicate code.


otherwise I would suggest replacing this with the type `Union{BeerLambertModel, TwoStreamModel}`